MutableStorageKVP/.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "src/cJSON"]
```

Comment:
Shouldn't .gitmodules be at the root of the repository? I don't think this will work.